Configuration/Skimming/python/autoSkim.py

```diff
@@ -41,6 +41,10 @@
  'SingleMuon': 'LogError+LogErrorMonitor',
 }
 
+# For 2023 PbPb skims
+for i_split in range(32):
+    autoSkim[f'HIPhysicsRawPrime{i_split}'] = 'PbPbEMu+PbPbZEE+PbPbZMM+LogError+LogErrorMonitor'
```

Comment:
Maybe worth mentioning the following.

 - #42795 and #42796 contain the latest version of the HLT menus (incl. the "2023-V1.0" HIon menu).
 - It looks like the [`PbPbEMu`](https://github.com/cms-sw/cmssw/blob/a2fe5aab0bfb328573d0d080b45d1c3ad38f2f01/Configuration/Skimming/python/PbPb_EMuSkim_cff.py#L6) and [`PbPbZMM`](https://github.com/cms-sw/cmssw/blob/a2fe5aab0bfb328573d0d080b45d1c3ad38f2f01/Configuration/Skimming/python/PbPb_ZMMSkim_cff.py#L6) skims use `"HLT_HIL3SingleMu*_Open_v*"` as trigger requirement, and in the latest HIon menu that pattern matches only one single prescaled trigger, since other `HLT_HIL3SingleMu*_Open_v*` triggers were renamed in [CMSHLT-2882 (comment)](https://its.cern.ch/jira/browse/CMSHLT-2882?focusedId=4935731&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4935731).

Reply:
@mandrenguyen

Reply:
Dener or Soohwhan will comment

Reply:
@youyingli, to clarify, we had to change the L1 seeds for these triggers so to make sense we changed their names from "HLT_HIL3SingleMu*_Open_v*" to "HLT_HIL3SingleMu*_v*". Therefore the requirement strings need to be adjusted accordingly. Sorry this was not notified in advance. Thank you.

Reply:
Thanks!  Let me quickly update them in this PR.